src/features/pastelStatistics/components/card/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Card'
```

Comment:
I don't think that this is necessary. You should export this component from the feauture `index` file